apps/rush-lib/src/api/LastInstallFlag.ts

```diff
@@ -4,6 +4,141 @@ import { FileSystem, JsonFile, JsonObject } from '@microsoft/node-core-library';
 
 export const LAST_INSTALL_FLAG_FILE_NAME: string = 'last-install.flag';
 
+/** @internal */
```

Comment:
`@internal` is only needed if this is exported from the project's `index.ts`

Reply:
I see.  I've gone ahead and removed the extraneous `@internal` markers